Formula/wine.rb

```diff
@@ -65,49 +52,406 @@ def require_universal_deps?
   end
 
   resource "gecko" do
-    url "https://downloads.sourceforge.net/wine/wine_gecko-2.40-x86.msi", :using => :nounzip
-    sha256 "1a29d17435a52b7663cea6f30a0771f74097962b07031947719bb7b46057d302"
+    url "https://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86.msi", :using => :nounzip
+    sha256 "3b8a361f5d63952d21caafd74e849a774994822fb96c5922b01d554f1677643a"
   end
```

Comment:
Perhaps because no one bothered to figure out why the 64 bit version of the web engine was missing? 🤷‍♂️